sandy/core/cov.py

```diff
@@ -509,29 +509,36 @@ def invert(self, rows=None):
         M_inv = M_inv.reindex(index=index, columns=columns).fillna(0)
         return self.__class__(M_inv)
 
-    def sampling(self, nsmp, seed=None, rows=None, pdf='normal',
-                 tolerance=None, threshold=None):
+    def get_perturbation_coeff(self, nsmp, seed=None, rows=None, pdf='normal',

```

Comment:
We will call it sampling or get_samples. This is consistent with standard nomenclature. 
Perturbations will get mixed up with `custom_perturbation`.
These are anyway samples!